docs/reference-guides/data/data-core-edit-site.md

```diff
@@ -408,6 +408,10 @@ _Returns_
 
 Undocumented declaration.
 
+### toggleDistractionFree
+
+Action that toggles distraction free mode. DFM expects there are no sidebars, as due to the z-index values set, you can't close sidebars.
```

Comment:
```suggestion
Action that toggles distraction free mode. Distraction free mode expects there are no sidebars, as due to the z-index values set, you can't close sidebars.
```

The term "DFM" may be difficult to convey to other developers, as my experience is that it is an abbreviation within Gutenberg development. What do you think?